sc-memory/sc-memory/scs/scs.g4

```diff
@@ -51,9 +51,16 @@ content returns [ElementHandle handle]
   : ('_' { $ctx->isVar = true; } )?
       c=CONTENT_BODY
     {
-      std::string const v = $ctx->c->getText();
+      std::string v = $ctx->c->getText();
       SC_ASSERT(v.size() > 1, ());
-      $ctx->handle = m_parser->ProcessContent(v.substr(1, v.size() - 2), $ctx->isVar);
+      if (v.find('!') == 0)
```

Comment:
Please, check it in ProcessContent, don't pass any other paramaters to this method